vulnerabilities/unchecked-return-values.md

```diff
@@ -0,0 +1,58 @@
+### Unchecked Return Values
+
+The main idea behind this type of vulnerability is the failure to properly handle the return values of external function calls. This can have significant consequences, including fund loss and unexpected behavior in the contract's logic.
+
+In Solidity, developers can perform external calls using methods like:
+
+1. `.send()`
+2. `.call()`
+3. `.transfer()`
+
+`.transfer()` is commonly used to send ether to external accounts, however, the `.send()` function can also be used. For more versatile external calls, the `CALL` opcode can be directly employed in Solidity.
```

Comment:
Instead of

"For more versatile external calls, the CALL opcode can be directly employed in Solidity."

I think it would be more accurate and understandable to say

"For more versatile external calls, .call() can be used."